openapi-v1.yaml

```diff
@@ -1511,6 +1527,25 @@ definitions:
         items:
           $ref: "#/definitions/Pricing"
 
+  FileProperty:
```

Comment:
Do we still need this object?

Reply:
No, just needed to generate the rest model